Macro Z_ARC_MPU_SIZE_ALIGN seems to be missing

$ grep -Rn Z_ARC_MPU_SIZE_ALIGN *                                  
lib/libc/newlib/libc-hooks.c:54:#define HEAP_BASE	Z_ARC_MPU_SIZE_ALIGN(USED_RAM_END_ADDR)

```
/workdir/zephyr/lib/libc/newlib/libc-hooks.c: In function 'malloc_prepare':
/workdir/zephyr/lib/libc/newlib/libc-hooks.c:54:19: error: implicit declaration of function 'Z_ARC_MPU_SIZE_ALIGN'; did you mean 'Z_ARC_MPU_ALIGN'? [-Werror=implicit-function-declaration]
   54 | #define HEAP_BASE Z_ARC_MPU_SIZE_ALIGN(USED_RAM_END_ADDR)
     |                   ^~~~~~~~~~~~~~~~~~~~
/workdir/zephyr/lib/libc/newlib/libc-hooks.c:79:29: note: in expansion of macro 'HEAP_BASE'
   79 |  z_malloc_partition.start = HEAP_BASE;
     |                             ^~~~~~~~~
```

Tests: tests/lib/mem_alloc/
Board: em_starterkit_em7d
Config: prj_newlib.conf 


Found out this in one of my PR, where for some reason CI dared to run the right test which trigger a build error.
I don't know why CI could not catch this earlier.

--- a/lib/libc/newlib/libc-hooks.c
+++ b/lib/libc/newlib/libc-hooks.c
@@ -51,7 +51,7 @@ MALLOC_BSS static unsigned char __aligned(CONFIG_NEWLIB_LIBC_ALIGNED_HEAP_SIZE)
 #define HEAP_BASE	ROUND_UP(USED_RAM_END_ADDR, \
 				 CONFIG_ARM_MPU_REGION_MIN_ALIGN_AND_SIZE)
 #elif defined(CONFIG_ARC)
-#define HEAP_BASE	Z_ARC_MPU_SIZE_ALIGN(USED_RAM_END_ADDR)
+#define HEAP_BASE	ROUND_UP(USED_RAM_END_ADDR, Z_ARC_MPU_ALIGN)
 #else
 #error "Unsupported platform"
 #endif /* CONFIG_<arch> */

